### DISCUSSION TITLE
Can I change NERDTreeMenuDown & NERDTreeMenuUp because I use colemak keyboard lay out.

### DISCUSSION THREAD

**Discussion by steven130169** [JetBrains/ideavim#628] [Q&A]
I use this code into my .ideavimrc is not workable for me. Have another method to change the keymap?
```vim
let g:NERDTreeMenuDown = 'n'
let g:NERDTreeMenuUp = 'e'
```

**Comment by lippfi** [comment]
It is not possible to configure the NERDTree plugin via .ideavimrc yet. The only option I see is to fork the IdeaVim repository and change the keymap manually in the following file: https://github.com/JetBrains/ideavim/blob/master/src/main/java/com/maddyhome/idea/vim/extension/nerdtree/NerdTree.kt
